src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -519,28 +558,19 @@ const ClinicalAnalysisResult = ({
                 ],
               };
 
-              const ContinuousWrapper = withFacetData(props => {
-                const facets = JSON.parse(props.viewer.explore.cases.facets);
+              if (varProperties.plotTypes === 'continuous') {
                 return (
                   <ContinuousAggregation
+                    key={varFieldName}
                     fieldName={varFieldName}
-                    stats={facets[varFieldName].stats}
+                    stats={parsedFacets[varFieldName].stats}
+                    hits={hits}
                     filters={filters}
                     variable={varProperties}
                     plots={plotTypes[varProperties.plotTypes || 'categorical']}
                     style={{ minWidth: controlPanelExpanded ? 310 : 290 }}
                     id={id}
```

Comment:
Does ContinuousAggregation use id and plots?

Reply:
yup...for the ClinicalVariabledCard component